apps/documentation/src/routes/documentation/tutorial/api-fetching.mdx

```diff
@@ -13,11 +13,11 @@
 
 # API fetching
 
-The goal is to use the [PokeAPI](https://pokeapi.co/docs/v2) to list all the pokemons of the first generation (the best one btw) and then reserve a \_\_ page for each one separately.
+The goal is to use the [PokeAPI](https://pokeapi.co/docs/v2) to list all the Pokémon of the first generation (the best one, btw) and then reserve a dynamic page for each one separately.
 
-## Fetch all the pokemons
+## Fetch all the Pokémon
 
-To start let’s fetch all of them in the root page; since we want to render them on the server side we gonna need to implement the logic in the `index.rs` file.
+To start, let’s fetch all of them from the root page. Since we want to render them on the server side, we going to need to implement the logic in the `index.rs` file.
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>An auxiliary verb seems to be missing from this progressive structure. Did you mean “we're going”, “we are going”, or “we were going”? (PRP_VBG[13])
Suggestions: `we're going`, `we are going`, `we were going`
Rule: https://community.languagetool.org/rule/show/PRP_VBG?lang=en-US&subId=13
Category: GRAMMAR
<!-- __reviewdog__:Cg8yYjliNjU5MGFmZGFkMDASDExhbmd1YWdlVG9vbA== -->